agreements/employment.md

```diff
@@ -0,0 +1,167 @@
+
+6. ### Fixed part-time staff
+    1. If your pay is specified as being "fixed part time" then you will work for a number days per week to be agreed at the commencement of this contract.
+    2. You will be paid pro rata as follows: the number of days you work each week will be divided by five, the resulting fraction will be multiplied by one twelfth of the equivalent full time annual salary to give the amount of pay you are entitled to each month.
+
+7. ### Probationary period
+    1. You will be subject to an initial four month probationary period.
+    2. If at the conclusion of your probationary period the company requires more time to assess your performance, your probationary period may be extended by a further two months.
+
+8. ### Notice period
+    1. In cases of gross misconduct, you may be dismissed without notice. The following is an indicative, non-exhaustive list of things that might constitute gross misconduct:
+        1. insubordination;
+        2. committing a tort or being convicted for a criminal offence that impacts on your employment;
+        3. fighting or assaulting another person or causing another person to fear violence;
+        3. deliberately damaging property;
+        4. dishonesty, theft or fraud;
+        5. deliberately or recklessly disclosing confidential information or personal data to a third party;
+        6. willfully or seriously breaching the terms of this contract.
+    2. Other than in cases of gross misconduct, your notice period:
+        1. during your probationary period is two weeks;
+        2. following your probationary period up to the second anniversary of your commencement date is six weeks;
+        3. after the second anniversary of your commencement date, six weeks plus an additional week for each year of service up to a maximum of 12 weeks.
+    3. These periods apply to both you and the company, irrespective of who is giving notice to whom.
+    4. Notice to quit will only be effective if given in writing, which includes by e-mail.
+    5. Once notice has been given (whether from you to us, or from us to you), we reserve the right, at any time after the notice was given while you are still employed with us, to give you pay in lieu of notice. This means that we will pay you, as a lump sum, the salary that would otherwise be payable to you during the unexpired portion of your notice period.
+
+9. ### Holiday entitlement
+    1. We operate a holiday year which runs from 1st July to 30th June in the following year.
+    2. Leave accrues throughout the year according to the formula given below, resulting in 33 days paid leave for full time staff and a pro rata proportion for those working part time.
```

Comment:
Just an FYI on this, because it confuses lots and lots of people: First, there really isn't a "most companies". Trust me on this. There is a huge variation over what people actually do. 

Second, the number of bank and public holidays may vary during a year (eg with a special holiday for a jubilee or something). Some employers prefer to give employees a (generous) holiday allowance and know in advance exactly how many days of holiday there are in a year, rather than it changing between years depending on the whim of the government. Equally other companies don't do it that way.

Third, bank holidays and public holidays have almost no legal effect. Some companies choose to take them as holidays. I am not sure what dxw does. If there is a tradition of everyone taking public holidays as annual leave then it makes more sense to figure them in rather than having a fixed figure. But it depends a bit on company culture.

My own preference (from having drafting and litigated many many of these things) is just not to have bank and public holidays and keep things simple with N days per year.

To confuse a teeny bit more, the law requires 5.6 *weeks* holiday in any year to be paid. That would be 28 days for someone working full time (5/7). But the way in which that sort of leave is worked out is measured in weeks not days.